lib/capybara/queries/selector_query.rb

```diff
@@ -494,6 +497,12 @@ def matches_class_filter?(node)
         end
       end
 
+      def matches_focused_filter?(node)
```

Comment:
This should return true if no focused option was passed -- see `matches_id_filter?` and `use_default_id_filter?`